packages/sysutils/eventlircd/evmap/osmc_rf2.evmap

```diff
@@ -0,0 +1,13 @@
+#KEY_HOME		= KEY_HOME
+KEY_I			= KEY_INFO
+#KEY_UP			= KEY_UP
```

Comment:
indentation on = KEY_UP (cosmetic)

Reply:
duplicated #130 and just changed buttons

cat -A on raw shows it's OK.